net.sf.eclipsecs.core/src/net/sf/eclipsecs/core/projectconfig/filters/FilesOlderThanOneDayFilter.java

```diff
@@ -0,0 +1,21 @@
+package net.sf.eclipsecs.core.projectconfig.filters;
+
+import org.eclipse.core.resources.IFile;
+
+public class FilesOlderThanOneDayFilter extends AbstractFilter {
+
+	private static final long MILLIS_IN_24_HOURS = 1000 * 60 * 60 * 24;
+
+	@Override
+	public boolean accept(Object o) {
+
+		if (o instanceof IFile) {
+			IFile file = (IFile) o;
+			if ((System.currentTimeMillis() - file.getLocalTimeStamp()) < MILLIS_IN_24_HOURS) {
+				return true;
+			}
+		}
+		return false;

```

Comment:
All other filter implementations let unknown stuff pass (i.e. they return true by default). Your filter should behave the same and only return false at the end of the instanceof block. Otherwise if there are non-file objects, your filter always rejects them.